File: TransCoder_tokenized_test_set_functions/java/FIND_ROW_NUMBER_BINARY_MATRIX_MAXIMUM_NUMBER_1S.java
static void findMax ( int arr [ ] [ ] ) {
  int row = 0, i, j;
  for ( i = 0, j = N - 1;
  i < N;
  i ++ ) {
    while ( j >= 0 && arr [ i ] [ j ] == 1 ) {
      row = i;
      j --;
    }
  }
  System . out . print ( "Row number = " + ( row + 1 ) );
  System . out . print ( ", MaxCount = " + ( N - 1 - j ) );
}

File: TransCoder_tokenized_test_set_functions/cpp/FIND_ROW_NUMBER_BINARY_MATRIX_MAXIMUM_NUMBER_1S.cpp
void findMax ( int arr [ ] [ N ] ) {
  int row = 0, i, j;
  for ( i = 0, j = N - 1;
  i < N;
  i ++ ) {
    while ( arr [ i ] [ j ] == 1 && j >= 0 ) {
      row = i;
      j --;
    }
  }
  cout << "Row number = " << row + 1;
  cout << ", MaxCount = " << N - 1 - j;
}